requirements.txt

```diff
@@ -9,4 +9,8 @@ pytest-cov==2.5.1
 hypothesis==6.14.5
 globus-sdk==1.6.1
 python-dotenv==0.14.0
-frozendict==2.0.3
\ No newline at end of file
+frozendict==2.0.3
```

Comment:
Should we use a later version of frozendict or any of these others?

Reply:
The frozendict version is less than a month old, and the others should be up to date as of a couple days ago, so I think we're ok